checkstyle.properties

```diff
@@ -1,12 +1,24 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16
-publicVersions=4.17
+archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18
+publicVersions=4.19
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+4.19.description=Upgrade to Checkstyle 8.19
+4.19.sqVersion=[6.7,7.7+]
+4.19.date=2019-04-29
+4.19.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.19
+4.19.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.19/checkstyle-sonar-plugin-4.19.jar
+
+4.18.description=Upgrade to Checkstyle 8.18
```

Comment:
It's entirely pointless to retroactively add this version. The point of retaining archived versions in these files is to the Marketplace recognizes them as valid versions of the plugin and offers an upgrade to the _current_ version. Since this version goes straight into the archive, no one will have it installed and so there's no need to upgrade _from_ it.

Reply:
Thank you for pointing this out. :)
However, `4.18` contained some fixes for SQ 7.6 and, afaik, people installed it manually - hence, I'd like to keep it as an archived version.